opentech/static_src/src/app/src/containers/GroupByRoundDetailView.js

```diff
@@ -45,13 +46,11 @@ class GroupByRoundDetailView extends React.Component {
 }
 
 const mapStateToProps = (state, ownProps) => ({
-    isErrored: getRoundsErrored(state),
-    errorMessage: getByGivenStatusesError(ownProps.submissionStatuses)(state),
+    isErrored: getRoundsErrored(state) || getByStatusesError(state),
     isLoading: (
-        getByGivenStatusesLoading(ownProps.submissionStatuses)(state) ||
-        getRoundsFetching(state)
+        getByStatusesLoading(state) || getRoundsFetching(state)
     ),
-    submissions: getCurrentRoundSubmissions(state),
+    submissions: getCurrentStatusesSubmissions(state),
```

Comment:
@tm-kn This is the only functional change. I think this change is correct... It seemed wrong that we were fetching round submissions when we are interested in the status submissions. (I guess because they are a similar thing here it was working before..)